proposals/p4880.md

```diff
@@ -0,0 +1,141 @@
+
+Looking in more detail at our key results for 2024:
+
+-   [Carbon's toolchain implements enough of the language to build realistic code](https://github.com/carbon-language/carbon-lang/blob/10189bbb78db7b143a6d9d62797fc9698363fe4d/docs/project/roadmap.md#carbons-toolchain-implements-enough-of-the-language-to-build-realistic-code)
+    -   A smashing success, overall.
+    -   We have imports, a working prelude, generic types and functions.
+    -   We have many of the building blocks of dispatching through interfaces
+        for operator overloading, but there are still a few gaps left. Despite
+        the gaps, we're actually using our generic building blocks effectively,
+        including with a working `Core.Int` generic integer type that backs
+        `i32` instead of a hard coded type!
+    -   We didn't get to the more stretch parts of this goal like templates
+        though.
+-   [Carbon's toolchain can build C++ code](https://github.com/carbon-language/carbon-lang/blob/10189bbb78db7b143a6d9d62797fc9698363fe4d/docs/project/roadmap.md#carbons-toolchain-can-build-c-code)
+    -   We have Clang integrated into the toolchain!
+    -   It works reasonably well for C++ code without system `#include`s.
+    -   But we still need to get some critical headers and other data that Clang
+        depends on to get this fully working with system `#include`s.
+-   [Carbon's toolchain works with existing, simple C++ build systems](https://github.com/carbon-language/carbon-lang/blob/10189bbb78db7b143a6d9d62797fc9698363fe4d/docs/project/roadmap.md#carbons-toolchain-works-with-existing-simple-c-build-systems)
+    -   Until we have all the system `#include` support work done, we can't drop
+        our toolchain into a C++ build system. =/ This one ended up largely not
+        landing this year.
+    -   We did end up integrating Carbon's toolchain into very simple Bazel
+        build rules that we use to continuously build and test a collection of
+        example Carbon code. This made sure the compilation model does work in a
+        realistic toolchain situation for Carbon cod.
+-   [Carbon has a design and toolchain implementation of basic C++ interop](https://github.com/carbon-language/carbon-lang/blob/10189bbb78db7b143a6d9d62797fc9698363fe4d/docs/project/roadmap.md#carbon-has-a-design-and-toolchain-implementation-of-basic-c-interop)
+    -   This was too ambitious of a goal, but we actually got very close. The
+        first importing of C++ headers (but without doing anything) has already
+        landed, so we weren't too far away from the most basic parts of this.
+-   [Give talks at 2-3 conferences covering 3-4 different Carbon topics](https://github.com/carbon-language/carbon-lang/blob/10189bbb78db7b143a6d9d62797fc9698363fe4d/docs/project/roadmap.md#give-talks-at-2-3-conferences-covering-3-4-different-carbon-topics)
+    -   We had a really strong year delivering talks at both C++ and LLVM
```

Comment:
Maybe include a number here, given there was a number in the goal? We gave 5 talks on different topics, plus one panel session.

Reply:
Done.